emperor/support_files/js/controller.js

```diff
@@ -262,12 +263,40 @@ define([
       selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
       trigger: 'none',
       items: {
+        'saveState': {
+          name: 'Save Emperor setup',
+          icon: 'edit',
+          callback: function(key, opts) {
```

Comment:
I believe this can be changed to be:

```
callback: scope.saveConfig
```

No need to create a "wrapper" function.